src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,59 @@
+import { useAppDispatch, useAppSelector } from '@/hooks/common';
+import { Notification as HDSNotification } from 'hds-react/components/Notification';
+import { RootState } from '@/store';
+import { FC } from 'react';
+import { clearNotification } from '@/reducers/notificationSlice';
+import { useTranslation } from 'react-i18next';
+
```

Comment:
Should notifications disappear after while automatically?

Reply:
There is an optional "autoClose" prop with hds react https://hds.hel.fi/components/notification/code, should we add this option to the notification reducer, or to all notifications by default? Should this be implemented here or added to the backlog?